backend/homework/1/data/src/main/java/org/kdu/JavaHandOn1/StudentRepository.java

```diff
@@ -0,0 +1,125 @@
+package org.kdu.JavaHandOn1;
+
+import java.util.ArrayList;
+import java.util.Objects;
+
+public class StudentRepository {
+    private static final ArrayList<Student> StudentList = new ArrayList<>();
+    loggingSystem ls = new loggingSystem();
+
+    /**
+     * @param id INTEGER --> unique student id
+     * @return True --> if id is already present
+     *          False --> if id is not present
+     */
+    private boolean checkIdAlreadyPresent(int id)
```

Comment:
This is a good example for S in SOLID principle make sure you try to follow this whenever trying to design functions or refactoring them.